Code/GraphMol/MolAlign/testMolAlign.cpp

```diff
@@ -384,8 +398,9 @@ void testMMFFO3AConstraints() {
   TEST_ASSERT(o3a);
   o3a->align();
   delete o3a;
-  double d = (m2.getConformer().getAtomPos(cIdx) -
-              m1->getConformer().getAtomPos(cIdx)).length();
+  double d =
+      (m2.getConformer().getAtomPos(cIdx) - m1->getConformer().getAtomPos(cIdx))
```

Comment:
Did you format this file? If yes, my linewidth is probably set wrongly. What do you use?